sympy/plotting/plot.py

```diff
@@ -1730,6 +1730,26 @@ def plot3d_parametric_surface(*args, **kwargs):
     return plots
 
 
+def conplot(*args, **kwargs):
+    """
+    Draws contour plot of a given function.
+    """
+    args = list(map(sympify, args))
+    symbols = set()
+    show = kwargs.pop('show', True)
+    plot_expr = check_arguments(args, 1, 2)
+    symbols |= plot_expr[0].free_symbols
```

Comment:
And at any rate, shouldn't it be required for the symbols to be specified, since they need to be in order?